src/OpenSage.Game/Network/SkirmishGameSettings.cs

```diff
@@ -118,6 +121,30 @@ public void Persist(StatePersister reader)
             reader.PersistUInt32(ref _mapFileSize);
             reader.PersistUInt32(ref _unknownInt4);
             reader.PersistUInt32(ref _unknownInt5);
+
+            if (version >= 4)
+            {
+                ushort unknown6 = 0;
+                reader.PersistUInt16(ref unknown6);
+
+                reader.PersistObject(ref _startingCash);
+            }
+            else
+            {
+                // TODO: Set default starting cash.
+            }
+        }
+    }
+
+    internal struct Money : IPersistableObject
+    {
+        public uint Amount;
```

Comment:
This solves the above todo and is what I did personally (though I didn't have this in a dedicated object, didn't realize it was so good catch), up to you if this is the route you want to go though.
```suggestion
        public uint Amount = 10000u;
```

Reply:
Ah thanks. I've done it a slightly different way, using the value parsed from `GameData.ini` - https://github.com/OpenSAGE/OpenSAGE/commit/4d08c8071f6fa0cf281bda99ffb0a0e7236dd656

Reply:
ooh, this is way better, good call